Core.lua

```diff
@@ -674,12 +674,11 @@ function R:IsAttemptAllowed(item)
 	end
 
 	local _, _, _, _, _, _, _, _, _, LfgDungeonID = GetInstanceInfo()
-	print("Torghast Layer " .. LfgDungeonID)
 	if item.requiredDungeons[CONSTANTS.LFG_DUNGEON_IDS.TORGHAST_LAYER] then
-		if tContains(CONSTANTS.LFG_DUNGEON_IDS.TORGHAST_LAYER, LfgDungeonID) then
-			print("Toy drop tracking for Torghast Layer " .. LfgDungeonID)
+		if tContains(CONSTANTS.LFG_DUNGEON_IDS, LfgDungeonID) then
```

Comment:
Wouldn't this return `true` for every dungeon in the shared constants table? (e.g., Hallow's End dungeon etc)